openmdao/visualization/n2_viewer/tests/test_viewmodeldata.py

```diff
@@ -138,31 +183,117 @@ def test_model_viewer_has_correct_data_from_sqlite(self):
+                         "children": [{"name": "a", "type": "param", "dtype": "ndarray"},
+                                      {"name": "Area", "type": "param", "dtype": "ndarray"},
+                                      {"name": "rho", "type": "param", "dtype": "ndarray"},
+                                      {"name": "Vu", "type": "param", "dtype": "ndarray"},
+                                      {"name": "Vr", "type": "unknown", "implicit": false,
+                                       "dtype": "ndarray"},
+                                      {"name": "Vd", "type": "unknown", "implicit": false,
+                                       "dtype": "ndarray"},
+                                      {"name": "Ct", "type": "unknown", "implicit": false,
+                                       "dtype": "ndarray"},
+                                      {"name": "thrust", "type": "unknown", "implicit": false,
+                                       "dtype": "ndarray"},
+                                      {"name": "Cp", "type": "unknown", "implicit": false,
+                                       "dtype": "ndarray"},
+                                      {"name": "power", "type": "unknown", "implicit": false,
+                                       "dtype": "ndarray"}]}]}
+                    """
+                                             )
+        expected_pathnames_betz = json.loads('[]')
+        expected_conns_betz = json.loads("""
+                [{"src": "indeps.a", "tgt": "a_disk.a"}, {"src": "indeps.Area", "tgt": "a_disk.Area"},
+                 {"src": "indeps.rho", "tgt": "a_disk.rho"}, {"src": "indeps.Vu", "tgt": "a_disk.Vu"}]
+                """)
+        expected_abs2prom_betz = json.loads("""
+                    {"input": {"a_disk.a": "a", "a_disk.Area": "Area", "a_disk.rho": "rho", "a_disk.Vu": "Vu"},
+                    "output": {"indeps.a": "a", "indeps.Area": "Area", "indeps.rho": "rho", "indeps.Vu": "Vu",
+                    "a_disk.Vr": "a_disk.Vr", "a_disk.Vd": "a_disk.Vd", "a_disk.Ct": "a_disk.Ct",
+                    "a_disk.thrust": "a_disk.thrust", "a_disk.Cp": "a_disk.Cp",
+                    "a_disk.power": "a_disk.power"}}
+                    """)
+        expected_declare_partials_betz = json.loads("""
+                ["indeps.a > indeps.a", "indeps.Area > indeps.Area", "indeps.rho > indeps.rho", "indeps.Vu > indeps.Vu", "a_disk.Vr > a_disk.a", "a_disk.Vr > a_disk.Vu", "a_disk.Vd > a_disk.a", "a_disk.Ct > a_disk.a", "a_disk.thrust > a_disk.a", "a_disk.thrust > a_disk.Area", "a_disk.thrust > a_disk.rho", "a_disk.thrust > a_disk.Vu", "a_disk.Cp > a_disk.a", "a_disk.power > a_disk.a", "a_disk.power > a_disk.Area", "a_disk.power > a_disk.rho", "a_disk.power > a_disk.Vu", "a_disk.Vr > a_disk.Vr", "a_disk.Vd > a_disk.Vd", "a_disk.Ct > a_disk.Ct", "a_disk.thrust > a_disk.thrust", "a_disk.Cp > a_disk.Cp", "a_disk.power > a_disk.power"]
+        """)
```

Comment:
seems like we could remove a lot of these `json.loads`
I imagine doing all that loadsing really makes the test slow

Reply:
I checked and even the biggest json.loads takes only about 20 microseconds. Doing it this way was convenient when copying and pasting from the actual value.